frontend/cypress/integration/patients.spec.js

```diff
@@ -39,6 +40,56 @@ describe('Patient', () => {
     });
   });
 
+  describe('Create', () => {
+    it('should create an etudiant', () => {
+      cy.get('[data-test-id="add-new-etudiant"]')
+        .click();
+
+      cy.get('[data-test-id="etudiant-first-name-input"] > input')
+        .type('Damir');
+      cy.get('[data-test-id="etudiant-last-name-input"] > input')
+        .type('Sagadbekov');
+      cy.get('[data-test-id="etudiant-doctor-name-input"] > input')
+        .type('My doctor');
+      cy.get('[data-test-id="add-patient-date-input"]')
+        .click();
+
+      selectValidDateOfBirth();
+
+      cy.get('[data-test-id="save-etudiant-button"]')
+        .click();
+
+      cy.get('[data-test-id="etudiant-table"] tr')
+        .should('have.length', 7);
+      cy.get('[data-test-id="etudiant-row-missing-info"]')
+        .should('have.length', 3);
+      cy.get('[data-test-id="etudiant-row-complete-info"]')
+        .should('have.length', 3);
+
+      cy.get('[data-test-id="notification-success"] p')
+        .should(
+          'have.text',
+          'L\'étudiant Damir Sagadbekov a bien été créé. Vous pourrez renseigner les champs manquants plus tard en cliquant le bouton "Modifier" du patient.',
+        );
+    });
+
+    it('should not create an etudiant due to invalid date', () => {
+      cy.get('[data-test-id="add-new-etudiant"]')
+        .click();
+
+      cy.get('[data-test-id="etudiant-first-name-input"] > input')
+        .type('Wolve');
+      cy.get('[data-test-id="etudiant-last-name-input"] > input')
+        .type('Rine');
+      cy.get('[data-test-id="etudiant-doctor-name-input"] > input')
+        .type('Professeur Xavier');
+      cy.get('[data-test-id="add-patient-date-input"]')
+        .click();
+
+      selectInvalidDateOfBirth();
```

Comment:
the name is missleading, you are not selecting an invalid date, just checking that you cannot select it